mappings/net/minecraft/world/biome/source/util/MultiNoiseUtil.mapping

```diff
@@ -0,0 +1,123 @@
+CLASS net/minecraft/class_6452 net/minecraft/world/biome/source/util/MultiNoiseUtil
+	METHOD method_37620 createParameterConfig (F)Lnet/minecraft/class_6452$class_6454;
+		ARG 0 parameter
+	METHOD method_37621 createParameterConfig (FF)Lnet/minecraft/class_6452$class_6454;
+		ARG 0 parameter1
+		ARG 1 parameter2
+	METHOD method_37622 (FFFFFF)Lnet/minecraft/class_6452$class_6460;
+		ARG 0 temperatureNoise
+		ARG 1 humidityNoise
+		ARG 2 continentalnessNoise
+		ARG 3 erosionNoise
+		ARG 5 weirdnessNoise
+	METHOD method_37623 createMixedNoisePoint (FFFFFFF)Lnet/minecraft/class_6452$class_4762;
+		ARG 0 temperature
+		ARG 1 humidity
+		ARG 2 continentalness
+		ARG 3 erosion
+		ARG 4 depth
+		ARG 5 weirdness
+		ARG 6 offset
+	METHOD method_37625 createMixedNoisePoint (Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;Lnet/minecraft/class_6452$class_6454;F)Lnet/minecraft/class_6452$class_4762;
+		ARG 0 temperature
+		ARG 1 humidity
+		ARG 2 continentalness
+		ARG 3 erosion
+		ARG 4 depth
+		ARG 5 weirdness
+		ARG 6 offset
+	CLASS class_4762 MixedNoisePoint
```

Comment:
This is no longer a point but multidimensional as now each parameter accepts a range. Feel free to rename.

Reply:
What would be a good name then? Something like `MixedNoiseRange` doesn't work, because it isn't a range itself.

Reply:
Hmm, biome class calls this a `parameter` in the codec format, doesn't it?

Reply:
So something like `NoiseParameters`?

Reply:
I thought about this, and, technically, wouldn't it be a hypercube now? https://en.wikipedia.org/wiki/Hypercube

Reply:
Sure, a hypercube is a good name